tasks/elasticsearch-ssl.yml

```
@@ -25,6 +25,7 @@
     owner: "{{ es_user }}"
     group: "{{ es_group }}"
     mode: "640"
+    remote_src: "{{ es_ssl_remote_certs }}"
```

name should be more explanatory of the action

```suggestion
    remote_src: "{{ es_ssl_remote_src }}"
```